.github/workflows/build_webkit.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches:
       - master
 
+env:
```

Comment:
It's better to move them on the jobs or even better step level. So if a GitHub Action gets compromised, its only accessible in the specific Step.